### PR TITLE
Font rendering

### PR DESCRIPTION
**Pull request by marimeireles**

### CODE REVIEW

**Review by thelmaboamah — COMMENTED** [review]
This is so cool. I wanna read all the chapters so I read along with this one and wow, there's a lot to know about fonts and layout.

This works great on my machine. I left a few minor comments.

File: src/recurse_browser/text_formatting.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+
+class Layout:
+    """
+    Computes and stores the position of each character in text
+    as a tuple in display_list
+    """
+
+    def __init__(self, tokens):
+        self.tokens = tokens
```
[the remark this answers is not in the document]
Comment (marimeireles) [reply src/recurse_browser/text_formatting.py:68]: True, alright!

File: src/recurse_browser/text_formatting.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    def __repr__(self):
+        return "Text('{}')".format(self.text)
+
+
+class Tag:
+    def __init__(self, tag):
+        self.tag = tag
+
+    def __repr__(self):
+        return "Tag('{}')".format(self.tag)
```
[the remark this answers is not in the document]
Comment (marimeireles) [reply src/recurse_browser/text_formatting.py:20]: Is it okay if we turn this into an issue?

**Comment by marimeireles** [comment]
Thank you for the review! ❤️

File: src/recurse_browser/text_formatting.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (marimeireles) [reply src/recurse_browser/text_formatting.py:20]: I've created #15 to keep track of it.
I think it's fine to just do it in a new PR. :)

**Pull request merged**